### ISSUE TITLE
Inconsistency with ANSI escape codes between xterm and Windows Terminal

### ISSUE THREAD

**Issue by ChrisJefferson** [microsoft/terminal#11097]
### Windows Terminal version (or Windows build number)

1.9.1942.0

### Steps to reproduce

cat the following file (hopefully escape codes cut+paste correctly) in WSL2 in Windows Terminal

```
[?25l
[?1049hr
[?25h
[?1049l
```

EDIT: File also in a gist: https://gist.github.com/ChrisJefferson/f0fdf72ada8568df77faf171a403cc5e

### Expected Behavior

Cursor is visible (as it is in standard xterm)

### Actual Behavior

Cursor is invisible.

**Comment by ChrisJefferson** [comment]
This is a (greatly cut down) use of ncurses in GAP (www.gap-system.org), where a cursor was disappearing and not reappearing.

**Comment by zadjii-msft** [comment]
can you try with 1.10 or 1.11? I think this was fixed in 1.10, which just moved to Stable today.


I can't get this to repro any longer with either:
```
printf "\e[?25lfoo"; sleep 2; printf "\n\e[?25hbar" ; sleep 2 ; printf "\n"

printf "\e[?25l\e[?1049hfoo"; sleep 2; printf "\e[?1049l\n\e[?25hbar" ; sleep 2 ; printf "\n"
```

**Comment by ChrisJefferson** [comment]
Yes, fixed in latest preview (1.11). Thanks!
